### ISSUE TITLE
Normalize modalities using sum of squares function

### ISSUE THREAD

**Issue by withercp** [nih-megcore/MEGHACK_DTIMEG#6]
This may be helpful.

https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html#sklearn.preprocessing.normalize

**Comment by withercp** [comment]
Completed with 66b7169083afef71dbfafc3de8829e06ba59a17f
